packages/vm/typedoc.js

```diff
@@ -11,7 +10,6 @@ module.exports = {
     'lib/evm/**',
     'lib/state/cache.ts',
   ],
-  excludeNotExported: true,
```

Comment:
Do you have got a clear reason to have this removed? I haven't got the complete oversight of the consequences right now.

Reply:
yes this option was removed:

> This is mentioned in the release notes here: https://github.com/TypeStrong/typedoc/releases/tag/v0.20.0
> > Removed options - mode, excludeNotExported, includeDeclarations, ignoreCompilerErrors, entryPoint

Reply:
the real impacting change is that the experimental `library` mode was removed and there is a new default mode. so our newly regenerated docs after these changes may look quite different in structure and markup, but should hopefully be an improvement and align with the future development of the package.